### DISCUSSION TITLE
How to add more than one values in q-linear -progress

### DISCUSSION THREAD

**Discussion by bagladivyang03** [quasarframework/quasar#9350] [CLI - PWA mode] [answered]
Hello 
Actually, i wanted to make something like below

![image](https://user-images.githubusercontent.com/57760543/118951514-c5c26e80-b978-11eb-8117-b909ea83dffe.png)

I was trying to use the q-linear -progress quasar component but couldn't able to find anything to use many values
Can anyone pls help how it is possible to make this or any other quasar component?
It would be a great help !!

**Comment by bagladivyang03** [comment]
@metalsadman  can u pls help me with this .

**Comment by rstoenescu** [comment, marked as answer]
This is pretty basic to do without Quasar components. Example (with Quasar v2 since you haven't specified your version of Quasar, but the gist of it is the same): https://codepen.io/rstoenescu/pen/yLMgZRV

**Reply by bagladivyang03** [reply]
I m using the quasar v2

**Reply by bagladivyang03** [reply]
Thank u so much
